src/lib/pipeline.ml

```diff
@@ -392,6 +402,8 @@ let rec to_file_prefix:
       sprintf "%s-%s-%s" prev
         vc.Variant_caller.name
         vc.Variant_caller.configuration_name
+    | Rna_hla_typer s ->
+      sprintf "RNA_HLA_typer-%s" (to_file_prefix ?read s)
```

Comment:
We want the JSON and `Pipeline.Construct` (= what's user-level API) to be reasonably future-proof. We store those JSONs for later inspection.

The tag in the GADT decision should depend on whether we will have another HLA typer will the same I/O types as Seq2HLA; and as far as I understood, HLA typers don't have a standard output format.